python-sdk/tests/sql/operators/test_load_file.py

```diff
@@ -1048,3 +1048,28 @@ def test_aql_load_column_name_mixed_case_json_file_to_dbs(
 
     df = db.export_table_to_pandas_dataframe(test_table)
     assert df.shape == (3, 2)
+
+
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [
+        {
+            "database": Database.SQLITE,
+        },
+    ],
+    indirect=True,
+    ids=["Sqlite"],
+)
+@pytest.mark.parametrize("if_exists", ["replace", "append"])
+def test_tables_creation_if_they_dont_exist(database_table_fixture, if_exists):
+    """
+    Verify creation of new tables in case we pass if_exists=replace/append if they don't exists.
+    """
+    path = str(CWD) + "/../../data/homes_main.csv"
```

Comment:
We should have a constant which refers to `"/../../data/homes_main.csv"` , I see in multiple places we are referring to this path.

Reply:
This is quite contextual. Other places where this file is used is either a test for HTTP, s3 or GCS as file location.